ton_client/client/src/setup.rs

```diff
@@ -118,6 +115,13 @@ fn setup(context: &mut ClientContext, config: SetupParams) -> ApiResult<()> {
 
 
 #[cfg(not(feature = "node_interaction"))]
-fn setup(_context: &mut ClientContext, _config: SetupParams) -> ApiResult<()> {
+fn setup(context: &mut ClientContext, config: SetupParams) -> ApiResult<()> {
+    debug!("-> client.setup({:?})", config);
+
+    let internal_config = NodeClientConfig {
+        timeouts: Some((&config).into()),
```

Comment:
Can you use it without a reference here? I think it's possible because structure is moved inside `into()` anyway.

Reply:
Done